### PR TITLE
Move components ownership from Apps to Databases

### PR DESCRIPTION
**Pull request by hhorak**
@fila43 @pkubatrh FYI, feel free to double-check

### CODE REVIEW

File: configs/sst_cs_databases-unwanted.yaml [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-  name: Unwanted packages from CS Apps SST
-  description: Packages we do not want to ship for CS Apps SST
-  maintainer: sst_cs_apps
+  name: Unwanted packages from CS Databases SST
+  description: Packages we do not want to ship for CS Databases SST
+  maintainer: sst_cs_databases
   unwanted_packages:
-    - js-uglify
-    - web-assets-devel
     - mercurial # Hard to maintain while not the most popular choice anymore.
```
Comment (pkubatrh) [inline configs/sst_cs_databases-unwanted.yaml:8]: I guess this should be removed as well now that it is covered in [sst_cs_base_utils-unwanted.yaml](https://github.com/minimization/content-resolver-input/pull/1321/files#diff-0decb157f168bbd5a0470ef8635a676417526e1be45d4a38ac03e68b4b31ae28)?

Comment (hhorak) [reply configs/sst_cs_databases-unwanted.yaml:8]: yes, will fix

**Comment by ljavorsk** [comment]
Can we add `zlib-ng*` packages back to our subsystem as well? It was handed over to plumbers in [this commit](https://github.com/minimization/content-resolver-input/commit/492b57821f257a074a2b78e9b57a9e5e70c1efa7)

**Comment by hhorak** [comment]
> Can we add `zlib-ng*`

Done.

**Review by yselkowitz — APPROVED** [review]

**Pull request merged**